scripts/xtensa-build-zephyr.py

```diff
@@ -70,7 +70,6 @@
 	xtensa_tools_version_postfix = "-unsupportedOS"
 	warnings.warn(f"Your operating system: {py_platform.system()} is not supported")
 
-
```

Comment:
spurious change?

Reply:
No whitespace changes in this file please, it's fairly active and does not need extra git conflicts.

Reply:
fixed